src/test/java/uk/gov/companieshouse/accounts/user/integration/UsersServiceTest.java

```
@@ -142,10 +142,10 @@ public void after() {
 
     @Test
     void setRolesWithNullOrMalformedOrNonexistentUserIdUserDoesNothing(){
-        Assertions.assertThrows( RuntimeException.class, () -> usersService.setRoles( null, List.of( Role.SUPPORT_MEMBER ) ) );
-        Assertions.assertThrows( RuntimeException.class, () -> usersService.setRoles( "", List.of( Role.SUPPORT_MEMBER ) ) );
-        Assertions.assertThrows( RuntimeException.class, () -> usersService.setRoles( "$", List.of( Role.SUPPORT_MEMBER ) ) );
-        Assertions.assertThrows( RuntimeException.class, () -> usersService.setRoles( "999", List.of( Role.SUPPORT_MEMBER ) ) );
+        usersService.setRoles( null, List.of( Role.SUPPORT_MEMBER ) );
```

wrong test, ids should be mandatory and it should still throw error if not found. 404.